Tests/XCUITests/NavigationTest.swift

```diff
@@ -416,6 +416,9 @@ class NavigationTest: BaseTestCase {
 
         let addressBar = app.textFields["address"]
         XCTAssertTrue(addressBar.value(forKey: "hasKeyboardFocus") as? Bool ?? false)
+
+        // These instances are false positives of the swiftlint configuration
+        // swiftlint:disable empty_count
```

Comment:
Usually you have to enable the rule after the place where the warning was occurring otherwise you disable the rule the remaining lines in the file. Is that what you want?

Reply:
Yes. It's at the end of the file so I figured it was fine to leave it disabled. I can re-enable it if you think it's worthwhile. :)

Reply:
I'd re-enable it everywhere again where we disable it before. Otherwise if we add more test later we warnings that should be there. Basically saving my future self from not so pretty code 😉

Reply:
You could just use `// swiftlint:disable next empty_count` before each false positive or to keep the disabling with the line:
```
XCTAssert(app.keyboards.count > 0, "The keyboard is not shown") // swiftlint:disable this empty_count
```